e2e/testdata/fn-eval/missing-fn-image/.expected/config.yaml

```diff
@@ -17,5 +17,4 @@ exitCode: 1
 image: gcr.io/kpt-fn/dne # non-existing image
 args:
   namespace: staging
-stdErr: "Failed to fetch \"latest\" from request \"/v2/kpt-fn/dne/manifests/latest\""
-
+stdErr: 'failed to check function image existence: function image "gcr.io/kpt-fn/dne" doesn''t exist'
```

Comment:
Not this PR: Can we remove this `failed to check function image existence` part from the error ?

Reply:
We probably need to define a struct for this error?